src/ServiceFabricPubSub/Admin/Admin.csproj

```diff
@@ -20,6 +20,9 @@
     <PackageReference Include="Microsoft.ServiceFabric.Data" Version="2.6.204" />
```

Comment:
Please don't hesitate to ping the group for help on the port conflict issue. Its part of the upside of having a team to help solve problems. :)